### PR TITLE
New variable VERISIGN_PRODUCT_(authority.name)

### PR DESCRIPTION
**Pull request by sirferl**
If there is a config variable with VERISIGN_PRODUCT_<upper(authority.name)> take the value as Cert product-type else default to "Server" to be compatible with former versions.
This enables the creation of different Verisign cert-products eg. EV or Standard Certs using several cert authorities.

### CODE REVIEW

**Comment by sirferl** [comment]
@hosseinsh : Hi, I've  been a bit productive. Hope this can be added swiftly, it would help us with a situation. Thank you !

File: lemur/plugins/lemur_verisign/plugin.py [hunk 1]
```diff
@@ -98,10 +98,16 @@ def process_options(options):
     :param options:
     :return: dict or valid verisign options
     """
+    # if there is a config variable with VERISIGN_PRODUCT_<upper(authority.name)> take the value as Cert product-type
+    # else default to "Server", to be compatoible with former versions
+    authority = options.get("authority").name.upper()
+    product_type = current_app.config.get("VERISIGN_PRODUCT_{0}".format(authority))
```
Comment (hosseinsh) [inline lemur/plugins/lemur_verisign/plugin.py]: One minor suggestion, to leverage  the default  option, and avoid the following if-statement.
``product_type = current_app.config.get("VERISIGN_PRODUCT_{0}".format(authority), "Server")``

**Comment by hosseinsh** [comment]
hi @sirferl, thanks for the PR. I had only a minor suggestion. Otherwise it looks all good to me.

**Comment by sirferl** [comment]
@hosseinsh: Thank you for showing me the right way to do it. I added youe suggestion.

**Comment by hosseinsh** [comment]
Thanks. Just merged it.

**Pull request merged**